experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,341 @@
 {% extends "common/with_sidebar.html" %}
 
-{% block title %}
-  Summary
-{% endblock title %}
+{% load nimbus_extras %}
+
+{% block title %}Summary{% endblock %}
 
 {% block sidebar %}
   {% include "common/experiment_details_sidebar.html" with experiment=experiment %}
 
-{% endblock sidebar %}
+{% endblock %}
 
 {% block main_content %}
-  <div class="container-fluid">{{ experiment.slug }}</div>
-{% endblock main_content %}
+  <div class="container-fluid py-4">
+    <!-- Experiment Details Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Experiment Details</h4>
+      </div>
+      <div class="card-body">
+        <p>
+          <strong>Slug:</strong> {{ experiment.slug }}
+        </p>
+        <p>
+          <strong>Name:</strong> {{ experiment.name }}
+        </p>
+      </div>
+    </div>
+    <!-- Takeaways Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Takeaways</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th>QBR learning</th>
+              <td>{{ experiment.takeaways_qbr_learning }}</td>
+              <th>Statistically significant DAU Gain</th>
+              <td>{{ experiment.takeaways_metric_gain }}</td>
+            </tr>
+            <tr>
+              <th>Summary</th>
+              <td colspan="3">{{ experiment.takeaways_summary|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th>Gain amount</th>
+              <td colspan="3">{{ experiment.takeaways_gain_amount|format_not_set |safe }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Overview Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Overview</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th>Slug</th>
+              <td>{{ experiment.slug }}</td>
+              <th>Experiment owner</th>
+              <td>{{ experiment.owner }}</td>
+            </tr>
+            <tr>
+              <th>Application</th>
+              <td>{{ experiment.get_application_display }}</td>
+              <th>Public description</th>
+              <td>{{ experiment.public_description|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th>Feature config</th>
+              <td>
+                {% for feature in experiment.feature_configs.all %}
+                  {{ feature.name }}- {{ feature.description }}
+                {% empty %}
+                  <span style="color: red;">Not set</span>
```

Comment:
Oh for cases like this you'd wanna use the [danger](https://getbootstrap.com/docs/5.3/customize/color/#theme-colors) colour like

```html
<span class="text-danger">...</span>
```